src/App.tsx

```diff
@@ -3,12 +3,13 @@ import ErrorPage from "./pages/ErrorPage";
 import InsertJobs from "./pages/InsertJobs";
 import Layout from "./components/Layout";
 import Login from "./pages/Login";
+import Report from "./pages/Report";
 
 function App() {
   return (
     <Router>
       <Routes>
-        <Route index element={<Login />} />
+        <Route index element={<Report />} />
```

Comment:
A página de Login é o ponto principal da aplicação, o `index` route.
```suggestion
        <Route index element={<Login/>} />
```